src/WebUI/WorkLoads/WorkloadDetails.tsx

```diff
@@ -185,22 +214,34 @@ export class WorkloadDetails extends BaseComponent<IWorkloadDetailsProperties, I
         });
     }
 
-    private static _renderImageCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<any>, tableItem: any): JSX.Element {
+    private _renderImageCell = (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<any>, tableItem: any): JSX.Element => {
         let itemToRender: React.ReactNode;
         if (rowIndex == 0) {
             itemToRender = BaseKubeTable.defaultColumnRenderer(Resources.ImageText, "w-details-cell-header");
         }
         else {
-            // Todo :: Add support for making Image a link to Image Details view
-            itemToRender = <div className="w-details-cell-value">
-                {Utils.getImageText(tableItem.podTemplate.spec)}
-            </div>;
+            const podslist = this._podsStore.getState().podsList;
+            const pods: V1Pod[] = podslist && podslist.items || [];
+            const imageId = Utils.getImageId(Utils.getFirstImageName(tableItem.podTemplate.spec), tableItem.podTemplate.metadata, pods);
+            const imageName: string = Utils.getImageText(tableItem.podTemplate.spec);
+            // HardCoding hasImageDetails true for the time being, Should change it once we integrate with ImageService
+            //const hasImageDetails: boolean = this._imageDetailsStore.hasImageDetails(imageName);
+            const hasImageDetails = true;
+            itemToRender = hasImageDetails ?
+                (<Link
```

Comment:
please check my PR for this code or having a link in the column value.

Following is your code.
(<Link
                    className="fontSizeM text-ellipsis bolt-table-link bolt-table-inline-link w-details-cell-value"
                    onClick={() => this._showImageDetails(imageId)}>
                    {imageName || ""}
                </Link>) :
                (<div className="w-details-cell-value">
                    {imageName}
                </div>);

Reply:
Will revisit this once your PR is checked in